Tests/XiCliTests/XiCliTests.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  XiCliTests.swift
+//  XiCliTests
+//
+//  Created by Liam on 11/4/18.
+//  Copyright © 2018 Raph Levien. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+@testable import XiCli
+
+class CommandLineToolTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+        XCUIApplication().launch()
+        
+        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testExample() {
```

Comment:
nit: Empty, not used tests always make me sad :cry:

Reply:
Currently, the structure if the CLI is still in flux (thinking of switching to swiftpm Utilities) so I was planning to write the tests as soon as it became a little more concrete.